src/Tribe/Main.php

```diff
@@ -1263,7 +1263,10 @@ public function render_notice_archive_slug_conflict() {
 			$edit_settings_link = __( ' ask the site administrator to set a different Events URL slug.', 'the-events-calendar' );
 
 			if ( current_user_can( $settings_cap ) ) {
-				$setting_page_link  = tribe( Tribe\Events\Admin\Settings::class )->get_url() . '#tribe-field-eventsSlug';
+				$setting_page_link  = tribe( Tribe\Events\Admin\Settings::class )->get_url( [
```

Comment:
Since this line is changed anyway, it would be a good time to use an import for this class instead of the fully-qualified name.